examples/chat-gevent/chat.py

```diff
@@ -11,7 +11,11 @@
 
 @app.route('/')
 def index():
-    return render_template('index.html')
+    uri = '%s://%s/%s' % \
+        (request.environ['wsgi.url_scheme'] == 'https' and 'wss' or 'ws', \
```

Comment:
This should use a more readable syntax than inline `and` and `or`, e.g:

    scheme = 'wss' if (request.environ['wsgi.url_scheme'] == 'https') else 'ws'
    uri = '%s://%s/websocket' % (scheme, request.headers['host'])